org-formation/075-security-hub/security-hub.yaml

```diff
@@ -3,6 +3,10 @@ Description: "Setup AWS Security Hub"
 Resources:
   SecurityHub:
     Type: "AWS::SecurityHub::Hub"
+  SecurityHubStandard:
+    Type: "AWS::SecurityHub::Standard"
+    Properties:
+      StandardsArn: arn:aws:securityhub:us-east-1::standards/cis-aws-foundations-benchmark/v/1.4.0
```

Comment:
The examples from [the docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-securityhub-standard.html#aws-resource-securityhub-standard--examples) contain parameters for partition and region which we should probably follow..

```
StandardsArn: !Sub 'arn:${AWS::Partition}:securityhub:${AWS::Region}::standards/cis-aws-foundations-benchmark/v/1.4.0'
```